libs/Options/Customizer.php

```diff
@@ -881,11 +881,16 @@ public function customizer_style_overrides(){
 				<?php endif; ?>
 
 				<?php if ( get_theme_mod('nebula_hero_overlay_color') || get_theme_mod('nebula_hero_overlay_opacity') ): ?>
-					<?php $hero_overlay = ( get_theme_mod('nebula_hero_overlay_color') )? 'background: ' . get_theme_mod('nebula_hero_overlay_color') . ' ;' : ''; ?>
-					<?php $hero_overlay .= ( !is_null(get_theme_mod('nebula_hero_overlay_opacity')) )? 'opacity: ' . get_theme_mod('nebula_hero_overlay_opacity') . ';' : ''; ?>
-					<?php $hero_overlay .= 'animation: none;'; ?>
+					<?php if ( get_theme_mod('nebula_hero_overlay_color') ):?>
+						#hero-section .custom-color-overlay {background: <?php echo get_theme_mod('nebula_hero_overlay_color'); ?>;}
+					<?php endif;?>
 
-					#hero-section .custom-color-overlay{<?php echo $hero_overlay; ?>}
+					<?php if ( !is_null(get_theme_mod('nebula_hero_overlay_opacity')) ): ?> 
+						#hero-section .custom-color-overlay {opacity: <?php echo get_theme_mod('nebula_hero_overlay_opacity'); ?>;}
+					<?php endif;?>
+
+					#hero-section .custom-color-overlay{animation: none;}
+				
 				<?php endif; ?>
```

Comment:
Since the `custom-color-overlay` class is only active when the Customizer was used, we don't necessarily need extra conditionals here. I'm thinking it could be as simple as this one-liner:

```css
#hero-section .custom-color-overlay {background: <?php echo get_theme_mod('nebula_hero_overlay_color'); ?>; opacity: <?php echo get_theme_mod('nebula_hero_overlay_opacity'); ?>; animation: none;}
```